configure.ac

```diff
@@ -1475,7 +1475,22 @@ AC_ARG_ENABLE(omkafka,
          esac],
         [enable_omkafka=no]
 )
+AC_ARG_ENABLE(kafka_tests,
+        [AS_HELP_STRING([--enable-kafka-tests],[enable Kafka specific tests in testbench @<:@default=no@:>@])],
+        [case "${enableval}" in
+         yes) enable_kafka_tests="yes" ;;
+          no) enable_kafka_tests="no" ;;
+           *) AC_MSG_ERROR(bad value ${enableval} for --enable-kafka-tests) ;;
+         esac],
+        [enable_kafka_tests=no]
+)
+AM_CONDITIONAL(ENABLE_KAFKA_TESTS, test x$enable_kafka_tests = xyes)
+
 if test "x$enable_omkafka" = "xyes"; then
+	if test "x$enable_kafka_tests" = "xyes"; then
+		AX_PROG_JAVAC #we don't need javac, but macro documentation says JAVAC *must* be checked before JAVA
+		AX_PROG_JAVA
+	fi
```

Comment:
do we need this for the build (I don't think so, I think it built so far) or for the testbench test? If it is the latter, I think we should check it in the test case only, so that we keep folks able to build it on a machine that does not have the full runtime environment.